### PR TITLE
errors fixed

### PR DESCRIPTION
**Pull request by Temitopefa**
fixed errors on naming

### CODE REVIEW

File: src/components/Menus.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { makeStyles } from "@mui/styles";
+import { Avatar, Fade, Menu, MenuItem, Button } from '@mui/material'
+
+//images
+import { ZapiDash, ZapiApps, ZapiHelp, ZapiArrow, ZapiPic } from '../assets'
+
+interface Mobile {
```
Comment (pablo-clueless) [inline src/components/Menus.tsx:8]: If this is to check for screen size, why not just use media query and avoid this trouble??

**Review by pablo-clueless — COMMENTED** [review]
Please check for comments

File: src/assets/index.ts [hunk 2]
```diff
@@ -1,10 +1,17 @@
 import GoogleIcon from "./icons/GoogleIcon";
-import LinkedInIcon from "./images/Linkedin.png";
-import TwitterIcon from "./images/Twitter.png";
-import YoutubeIcon from "./images/Youtube.png";
+import LinkedInIcon from "./images/linkedin.png";
```
Comment (taiwoAkindele) [inline src/assets/index.ts:2]: This will affect production. Return the name to what it was before.

File: src/env.d.ts [hunk 3]
```diff
@@ -1,5 +1,4 @@
-
```
Comment (taiwoAkindele) [inline src/env.d.ts:1]: should we be able to see this?

**Review by taiwoAkindele — APPROVED** [review]

File: src/env.d.ts (hunk 3 above)
Comment (pablo-clueless) [inline src/env.d.ts:1]: This file should be removed, create a .env file in the root of your project directory and put these variables there.

**Review by pablo-clueless — CHANGES_REQUESTED** [review]
Delete env.d.ts and create an env file in the root of your project directory.

File: src/env.d.ts (hunk 3 above)
Comment (taiwoAkindele) [reply src/env.d.ts:1]: @Temitopefa

**Review by pablo-clueless — APPROVED** [review]

**Review by PhilemonBrain — APPROVED** [review]

**Pull request merged**